### PR TITLE
Add an option to report in seconds.

### PR DESCRIPTION
**Pull request by feserr**
Add the option to report in seconds to solve the issue #1075.

### CODE REVIEW

**Comment by feserr** [comment]
@googlebot I signed it!

**Comment by LebedevRI** [comment]
Thanks, i guess this looks about right.
This seems to be missing test coverage - are there no tests on how the other time units are actually printed to CSV/JSON?

**Comment by feserr** [comment]
> Thanks, i guess this looks about right.
> This seems to be missing test coverage - are there no tests on how the other time units are actually printed to CSV/JSON?

Almost all test are checking nanoseconds and there was another to check the CSV report in microseconds.

I have added to new test to check console, JSON and CSV output of every time option.

File: test/reporter_output_test.cc [hunk 1]
```diff
@@ -712,7 +799,7 @@ ADD_CASES(
 // ========================================================================= //
 // ------------------------- Testing StrEscape JSON ------------------------ //
 // ========================================================================= //
-#if 0 // enable when csv testing code correctly handles multi-line fields
+#if 0  // enable when csv testing code correctly handles multi-line fields
```
Comment (feserr) [inline test/reporter_output_test.cc:802]: I didn't notice this change, It have been change by the auto formatter. Should I left it as before?

Comment (LebedevRI) [reply test/reporter_output_test.cc:802]: I'd just leave it.

**Comment by LebedevRI** [comment]
Ok, thanks, i guess that is it.
Will merge once CI finishes.

**Review by LebedevRI — APPROVED** [review]

**Pull request merged**